src/expandableCalendar/index.tsx

```diff
@@ -255,29 +259,35 @@ const ExpandableCalendar = (props: ExpandableCalendarProps) => {
     }
   }, []);
 
-  useEffect(() => {
-    // date was changed from AgendaList, arrows or scroll
-    scrollToDate(date);
-  }, [date]);
-
   const handleScreenReaderStatus = (screenReaderEnabled: any) => {
     setScreenReaderEnabled(screenReaderEnabled);
   };
 
   /** Scroll */
+  const autoScroll = useRef(false);
+
+  const scrollToDate = throttle((date: string) => {
```

Comment:
Creating a throttled function in a function component is tricky. 
The "throttled" function is being recreated each render and basically losing the throttle feature. 
You will need to wrap this with useCallback to preserve the method.

Reply:
Done